content/about/quire.md

```diff
@@ -5,50 +5,31 @@ type: essay
 
-<div class="image-list">
+**Institutions currently publishing with Quire**:
 
-Our community members represent a broad spectrum of institutions, both large and small, providing vital scholarship in the arts and sciences. In addition to being used at Getty, the Quire community includes a growing number of scholars, curators, editors, designers, and web developers, and the digitally curious. Individuals at all levels of technical experience are welcome, from content creators to developers. We provide tools and resources to empower and educate our community.
+Getty / Hong Kong Museum and Art Gallery, Hong Kong University / Mills College Art Museum / Minneapolis Institute of Art / Tilt West / The Graphics Office / Leventhal Map &  Education Center, Boston Public Library / Ad Hoc Museum  Collective / San Jose Museum of Art / Nelson-Atkins Museum of Art / Pitts Theology Library, Emory University / Duke University / Northwestern University Libraries
 
-<div class="action-button">
-
-[**See What Our Community Is Creating**](/community/community-showcase/)
-</div>
-
-![Explore icon](/img/illustrations/undraw_Group_chat_unwm.png)
-
-</div>
-<div class="image-list">
-
-Quire is designed to ensure that digital books and their contents are not only as beautiful as the print books we know and love but are equally as enduring. Optimized for discoverability and longevity, Quire enables robust preservation and distribution of works in a simplified and accessible format.
-
-<div class="action-button">
-
-[**Learn How Quire Works**](/about/how-it-works/)
-</div>
-
-![Explore icon](/img/illustrations/undraw_in_progress_ql66.png)
+**Quire has been used to produce**:
 
-</div>
-<div class="image-list">
-
-Currently under development for potential release as [open-source software](/about/open-source/), Quire builds upon Getty’s expertise in traditional book publishing and is the cornerstone of the institution’s pioneering [digital publishing program](https://www.getty.edu/publications/digital/index.html). Through the development of Quire, Getty is carving a path forward for individuals and institutions looking to solve the challenges of creating sustainable and discoverable digital publications.
+Collection catalogues / exhibition catalogues / journals / online exhibitions / proceedings volumes / research projects / supplements to print publications / personal histories / guides and toolkits
 
 <div class="action-button">
 
-[**Join the Quire Community Today**](/community/join-us/)
-</div>
-
-![Explore icon](/img/illustrations/undraw_celebration_0jvk.png)
-
+[**How to Get Involved**](/community/join-us/)
 </div>
 
 ## Features & Functionality
 
+{{< q-figure-group id= "custom-styles, plain-text, video-embeds, collection-catalogue" class="is-pulled-right" >}}
```

Comment:
Good idea to add the images here, but I'd like to work more on the design so that they visually work better with the feature list and vice versa. It's pretty wonky now. Ok to remove the images until I can get to it?